doc/api/crypto.md

```diff
@@ -2958,6 +2958,171 @@ Does not perform any other validation checks on the certificate.
 
 ## `node:crypto` module methods and properties
 
+### `crypto.argon2(algorithm, parameters, callback)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.2 - Release candidate
+
+* `algorithm` {string} Variant of Argon2, one of `"argon2d"`, `"argon2i"` or `"argon2id"`.
+* `parameters` {Object}
+  * `message` {string|ArrayBuffer|Buffer|TypedArray|DataView} REQUIRED, this is the password for password
+    hashing applications of Argon2.
+  * `nonce` {string|ArrayBuffer|Buffer|TypedArray|DataView} REQUIRED, must be at
+    least 8 bytes long. This is the salt for password hashing applications of Argon2.
+  * `parallelism` {number} REQUIRED, degree of parallelism determines how many computational chains (lanes)
+    can be run. Must be greater than 1 and less than `2**24-1`.
+  * `tagLength` {number} REQUIRED, the length of the key to generate. Must be greater than 4 and
+    less than `2**32-1`.
+  * `memory` {number} REQUIRED, memory cost in 1KiB blocks. Must be greater than
+    `8 * parallelism` and less than `2**32-1`. The actual number of blocks is rounded
+    down to the nearest multiple of `4 * parallelism`.
+  * `passes` {number} REQUIRED, number of passes (iterations). Must be greater than 1 and less
+    than `2**32-1`.
+  * `secret` {string|ArrayBuffer|Buffer|TypedArray|DataView|undefined} OPTIONAL, Random additional input,
```

Comment:
I just found out about [WICG/webcrypto-modern-algos](https://github.com/WICG/webcrypto-modern-algos) and their specification for `Argon2Params` [uses `secretValue` instead of `secret`](https://wicg.github.io/webcrypto-modern-algos/#dfn-Argon2Params) Is that something we want to follow, so that the implementations are closer?

Reply:
Also, this implementation so far ignores the `version` param, assuming it will never be changed. v1.3 has been released and not changed since 2016 (v1.0 was released in 2015)

Reply:
Cool, that looks a lot like the generic KDF proposal that was mentioned very early in the history of this PR.

If you need a second pair of hands for anything let me know, I'm eager to participate 😄 

This PR is otherwise complete, I believe.